### DISCUSSION TITLE
umiMax左上角的图标大小和距离

### DISCUSSION THREAD

**Discussion by caokun19850706** [ant-design/ant-design#44796] [Q&A]
刚来学习，整了一天也没找到哪儿能自由改写左上角logo的样式，比如再大一点或者设置个border-radius，求指导

**Comment by poyiding** [comment]
1. 在 umiMax 项目中有个app.ts 目录，里面layout 可以配置logo和自定义logo, 相关的样式也可重写。
2. 这个layout 是依赖[https://procomponents.ant.design/components/layout](procomponents) 你可以在里面找相关的配置或者问题
3. umi 相关的问题可以去umi 项目那边提问或者自行google 查询

**Comment by caokun19850706** [comment]
非常感谢您的回复，layout配置在app.ts里可以找到颜色什么的，就margin不知道怎么配置，也没有说，能给个app.ts更改头像的margin示例吗？

**Reply by poyiding** [reply]
找不到token 就可以自定义样式或者重写样式

 比如在 global.less 里重写logo的样式。
.ant-pro-global-header-logo img { 
 margin: xxx
}

或者 logo: <img scr='xxx'  style={{ margin: xxxx }}  />
